docs/pbc/all/dynamic-multistore/202404.0/tutorials-and-howtos/how-to-use-store-timezone.md

```diff
@@ -0,0 +1,49 @@
+---
+title: How to use store context timezone in Yves.
+description: This guide shows how to use store context timezone in yves
+last_updated: Jun 5, 2024
+template: howto-guide-template
+---
+
+## Prerequisites
+
+* [Configure application context timezone](/docs/pbc/all/dynamic-multistore/{{page.version}}/tutorials-and-howtos/how-to-configure-application-timezone.html)
+
+
+## How to Use Store Context Timezone
+
+To use the store context timezone in Yves, you can follow these steps:
+
+1. Retrieve the store timezone using the `StoreTransfer`:
+Timezone is stored in the `StoreTransfer` object. You can retrieve the store timezone using the following code or something similar: 
+
+    ```php
+    $storeTransfer = $this->getFactory()->getStoreClient()->getCurrentStore();
+    $storeTimezone = $storeTransfer->getTimezone();
+    ```
+`StoreTransfer` will contain also store context collection property `StoreTransfer.applicationContextCollection` which contains the timezone for each application in the store.
```

Comment:
Timezone is already configured value that does not show customer how to use store context, please provide another example, e.g. with Date Format